packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -121,10 +121,10 @@ abstract class Route<T> {
   /// See [RouteSettings] for details.
   final RouteSettings settings;
 
-  /// The entries this route has placed in the overlay.
+  /// The overlay entries of this route.
   ///
-  /// These are typically populated by [install], added to the [Overlay] by the
-  /// [Navigator], and then removed by [dispose].
+  /// These are typically populated by [install], added to or removed from the
+  /// [Overlay] by the [Navigator]
   ///
   /// There must be at least one entry in this list after [install] has been
   /// invoked.
```

Comment:
I can't comment on that line, but you'll also have to update line 139-140 below. Also, the doc about `isInitialRoute` just below that line should be removed, I think?